dev_guide/jboss_tools.adoc

```diff
@@ -5,12 +5,15 @@
 :icons:
 :experimental:
 :toc: macro
-:toc-title: 
+:toc-title:
 
 toc::[]
 
 == Overview
 
-JBoss Tools is an umbrella project for a set of link:http://eclipse.org[Eclipse] plugins that supports JBoss and related technologies; 
+JBoss Tools is an umbrella project for a set of link:http://eclipse.org[Eclipse] plugins that supports JBoss and related technologies;
 there is support for OpenShift, Hibernate, JBoss AS, Drools, jBPM, JSF, (X)HTML, Seam, Maven, JBoss ESB, JBoss Portal and more.
```

Comment:
Alternatively, could also rework it to say "...a set of Eclipse plug-ins supporting JBoss and related technologies, including:" then turn the rest into a bulleted list? May be nicer to read.

Also note the minor nit "plug-ins" instead of "plugin", per Style Guide.